app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

```diff
@@ -5,7 +5,7 @@ description: |-
   Explore how you can get into teaching without a degree by taking an undergraduate course to gain qualified teacher status (QTS) alongside a degree.
 related_content:
     Is teaching right for me? : "/is-teaching-right-for-me"
-    5 reasons to attend a Get Into Teaching event : "/blog/5-reasons-to-attend-a-get-into-teaching-event"
+    Get Into Teaching events what to expect : "/events/about-get-into-teaching-events"
```

Comment:
I think we should maybe delete this Related content link altogether - not sure that the events offer much for people without a degree / not studying for one?